installation_and_upgrade/ibex_install_utils/check_config.py

```diff
@@ -0,0 +1,47 @@
+import os
+import tempfile
+import subprocess
+
+from git.repo import Repo
+from genie_python import genie as g
+
+from ibex_install_utils.user_prompt import UserPrompt
+
+
+PYTHON = os.path.join("C:\\", "Instrument", "Apps", "Python3", "genie_python.bat")
+CONFIG_CHECKER_REPO = "https://github.com/ISISComputingGroup/InstrumentChecker.git"
+CONFIGS_REPO = "http://spudulike@control-svcs.isis.cclrc.ac.uk/gitroot/instconfigs/inst.git"
+GUI_REPO = "https://github.com/ISISComputingGroup/ibex_gui.git"
+CONFIGS_DIR = "configs"
+GUI_DIR = "gui"
+TEST_REPORTS_DIR = "test-reports"
+
+
+def check_config():
```

Comment:
Is there a reason why this is implemented differently from every other task? I'd have thought it should be implemented in the same way as `perform_server_tests` for example, using the `@task` decorator and being called in a sensible place in `install_tasks.py`?

Currently this will run after the step to "inform instrument scientists" which isn't what I'd expect?